api/src/main/java/org/openmrs/module/idgen/service/db/IdentifierSourceDAO.java

```diff
@@ -142,6 +142,11 @@
      * @return the IdentifierSource with the given uuid
      */
     IdentifierSource getIdentifierSourceByUuid(String uuid);
+    
+    /**

```

Comment:
same here

Reply:
Yes this specific file is very well commented.